### ISSUE TITLE
Disable Weekends Darker Gray Background on Month View

### ISSUE THREAD

**Issue by nickian** [nextcloud/calendar#985]
I'm not sure if this happened on a recent update, but I noticed weekends have a gray background and weekdays are white when viewing a month. Can an option/toggle be added in the settings for this? I definitely prefer the days to all look the same.

The gray being used seems much too dark to me. I could tolerate a lighter gray, like #f7f7f7.

**Comment by My-Random-Thoughts** [comment]
I agree it's too dark.  If you install the "Custom CSS" app you can set it to whatever colour you like.  I set this:

```
#fullcalendar td.fc-day.fc-sat, #fullcalendar td.fc-day.fc-sun {
    background-color: #edf0f5;
}

```

**Comment by nickian** [comment]
Thanks for the suggestion! That works for now. I went ahead and just made it white again, haha.
